sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_delete_images_async.py

```diff
@@ -17,36 +17,51 @@
 
     Set the environment variables with your own values before running the sample:
     1) CONTAINERREGISTRY_ENDPOINT - The URL of you Container Registry account
-"""
 
+    This sample assumes your registry has at least four repositories.
+"""
 import asyncio
-from dotenv import find_dotenv, load_dotenv
 import os
-
+from dotenv import find_dotenv, load_dotenv
 from azure.containerregistry import ArtifactManifestOrder
 from azure.containerregistry.aio import ContainerRegistryClient
-from azure.identity.aio import DefaultAzureCredential
+from azure.containerregistry._helpers import _get_authority, _get_audience, _get_credential
```

Comment:
The samples shouldn't demonstrate use of private methods. These should probably just be in each sample.